<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayEditor</class>
 <widget class="QWidget" name="DayEditor">
  <property name="windowTitle"><string>Planificador Diario</string></property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridMeals">
     <!-- Breakfast -->
     <item row="0" column="0">
      <widget class="QGroupBox" name="grpBreakfast">
       <property name="title"><string>Desayuno</string></property>
       <layout class="QVBoxLayout">
        <item><widget class="QListWidget" name="lstBreakfast"/></item>
        <item>
         <layout class="QHBoxLayout">
          <item><widget class="QPushButton" name="btnAddBre"><property name="text"><string>Añadir</string></property></widget></item>
          <item><widget class="QPushButton" name="btnEditBre"><property name="text"><string>Editar</string></property></widget></item>
          <item><widget class="QPushButton" name="btnDelBre"><property name="text"><string>Eliminar</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>

     <!-- MidMorning -->
     <item row="0" column="1">
      <widget class="QGroupBox" name="grpMidmorning">
       <property name="title"><string>Media mañana</string></property>
       <layout class="QVBoxLayout">
        <item><widget class="QListWidget" name="lstMidmorning"/></item>
        <item>
         <layout class="QHBoxLayout">
          <item><widget class="QPushButton" name="btnAddMid"><property name="text"><string>Añadir</string></property></widget></item>
          <item><widget class="QPushButton" name="btnEditMid"><property name="text"><string>Editar</string></property></widget></item>
          <item><widget class="QPushButton" name="btnDelMid"><property name="text"><string>Eliminar</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>

     <!-- Lunch -->
     <item row="1" column="0">
      <widget class="QGroupBox" name="grpLunch">
       <property name="title"><string>Comida</string></property>
       <layout class="QVBoxLayout">
        <item><widget class="QListWidget" name="lstLunch"/></item>
        <item>
         <layout class="QHBoxLayout">
          <item><widget class="QPushButton" name="btnAddLun"><property name="text"><string>Añadir</string></property></widget></item>
          <item><widget class="QPushButton" name="btnEditLun"><property name="text"><string>Editar</string></property></widget></item>
          <item><widget class="QPushButton" name="btnDelLun"><property name="text"><string>Eliminar</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>

     <!-- Snack -->
     <item row="1" column="1">
      <widget class="QGroupBox" name="grpSnack">
       <property name="title"><string>Merienda</string></property>
       <layout class="QVBoxLayout">
        <item><widget class="QListWidget" name="lstSnack"/></item>
        <item>
         <layout class="QHBoxLayout">
          <item><widget class="QPushButton" name="btnAddSna"><property name="text"><string>Añadir</string></property></widget></item>
          <item><widget class="QPushButton" name="btnEditSna"><property name="text"><string>Editar</string></property></widget></item>
          <item><widget class="QPushButton" name="btnDelSna"><property name="text"><string>Eliminar</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>

     <!-- Dinner -->
     <item row="2" column="0" colspan="2">
      <widget class="QGroupBox" name="grpDinner">
       <property name="title"><string>Cena</string></property>
       <layout class="QVBoxLayout">
        <item><widget class="QListWidget" name="lstDinner"/></item>
        <item>
         <layout class="QHBoxLayout">
          <item><widget class="QPushButton" name="btnAddDin"><property name="text"><string>Añadir</string></property></widget></item>
          <item><widget class="QPushButton" name="btnEditDin"><property name="text"><string>Editar</string></property></widget></item>
          <item><widget class="QPushButton" name="btnDelDin"><property name="text"><string>Eliminar</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>

   <!-- General actions -->
   <item>
    <layout class="QHBoxLayout">
     <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
     <item><widget class="QPushButton" name="btnLoad"><property name="text"><string>Cargar…</string></property></widget></item>
     <item><widget class="QPushButton" name="btnSave"><property name="text"><string>Guardar…</string></property></widget></item>
     <item><widget class="QPushButton" name="btnExportPdf"><property name="text"><string>Exportar PDF</string></property></widget></item>
     <item><widget class="QPushButton" name="btnExportImg"><property name="text"><string>Exportar Imagen</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>